src/lib/isTwitterHandle.js

```diff
@@ -0,0 +1,5 @@
+const twitterHandle = /^(?=.*[a-zA-Z]|[0-9]*_+[0-9]*)[a-zA-Z0-9_]{5,15}$/;

```

Comment:
@veliyat please provide some link that shows that a minimum twitter handle is `5 characters`. Some cases say `4`, some said `3`. 
Also, the regex can be minimized to  
```javascript
^(?=.*[A-z]|[\d]*_+[\d]*)[\w_]{5,15}$
```
`\w` will match any alphanumeric, see a good [quick ref here](https://regexone.com/).

Also, as btw questions: 

1. Should we ensure that `@` appear as the first character? 
2. Even though it allows the use of underscore `_`, do we validate `_____` as a valid twitter handle? Just asking

Cc. @profnandaa.

Reply:
Thanks for the shorter regexp. Updated the same and tested.

About the twitter handle validation, I tried the validation on Twitter's actual form where it provided me a validation of characters between 5 to 15 hence I made the same here.

Here are the answers to your questions:
1. Not necessary, as far as I see, this can both be used to validate username and handle if we donot mandate the @ .
2. Yes _____ is a valid twitter handle. https://twitter.com/_____ is an actual user.

Reply:
Thanks for the explanation. 
My concern is that, 
**Are there any existing accounts that have 4 or fewer twitter handles**?

Reply:
`Not necessary, as far as I see, this can both be used to validate username and handle if we do not mandate the @ .`
This is actually true, but the main work for the regex was to validate twitter handle and not its username? 
That is why I was championing for the addition of `@` to make sure that it seems a valid handle. 
Not adding `@` means we are just validating username. 

May be we can hear thoughts from @profnandaa, @chriso or any other person.

Reply:
> **Are there any existing accounts that have 4 or fewer twitter handles**?

Yes I realize now that the older usernames are even for single characters.
https://twitter.com/_
https://twitter.com/a

I would make changes and push again.

Reply:
> May be we can hear thoughts from @profnandaa, @chriso or any other person.

Yes lets hear their comments and then take a decision on the same. Thanks!

Reply:
> I would make changes and push again.

Also there are usernames that contain only numbers as well. So virtually there are no restrictions except the alphanumeric and _. The only restriction that holds right now is 15 characters max.

Reply:
But, the current twitter validation on the Profile form is exactly the regexp that we have in code. So I am confused :)

Reply:
@veliyat, we can actually leave at `5-15` for now. 
Then, we wait for views from @profnandaa or @chriso or @U-4-E-A on the same.